src/i18n/en.json

```diff
@@ -66,6 +66,8 @@
     "drag_and_drop": "Drag and Drop to sort items.",
     "remove_warning": "Are you sure you want to delete this?",
     "edit_selected": "Edit Selected",
+    "copy_to_clipboard": "Copy to clipboard",
```

Comment:
this need to be parametrized
check parent ticket comments here
https://tipit.avaza.com/project/view/347915#!tab=task-pane&groupby=ProjectSection&view=vertical&task=3716377&stafilter=NotComplete&fileview=grid
Should say what you're copying.  So here it should say "Copy Order Number" instead of "Copy to clipboard"